CHANGES.rst

```diff
@@ -7,6 +7,10 @@ associations
 - Expand the sequence field in a file name for association files from
   3 characters to 5 characters. [#7061]
 
+- Moved text dump of associations to happen when using the '-D' option,
```

Comment:
Release 1.8.1 has just occurred. Hence, a rebase and moving this entry into the next unreleased section will need be done.